pysmt/optimization/goal.py

```diff
@@ -86,12 +88,13 @@ class MaximizationGoal(Goal):
     Attention: some Optimizer may not support this goal
```

Comment:
"Attention" used in this way is not English :smile: 

```suggestion
    Warning: some Optimizer classes may not support this goal
```